projects/packages/jetpack-mu-wpcom/tests/php/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad-test.php

```diff
@@ -128,47 +126,22 @@ public function test_update_checklist_temporary_dismissed_when_date_is_in_the_fu
 	}
 
 	/**
-	 * Test return the task list is dismissed when the date is in the past.
+	 * Test return not dismissed task list when the date is in the future.
 	 */
-	public function test_update_checklist_temporary_dismissed_when_date_is_in_the_past() {
```

Comment:
I moved this test to projects/packages/jetpack-mu-wpcom/tests/php/features/launchpad/class-launchpad-task-lists-test.php